installers/Installer.hs

```diff
@@ -27,12 +28,24 @@ detectCI = do
 
 main :: IO ()
 main = do
-  options' <- Config.options "Daedalus installer generator" Config.optionsParser
-  options  <- (\ci -> options' { Config.oCI = ci }) <$> detectCI
+  let os = case Sys.os of
+             "linux"   -> Linux
```

Comment:
I think this should be linux64

Reply:
Ditto

Reply:
@jmitchell should it be Linux64 as well? or should that latter part be Linux? Not sure if that's derived from `uname` or something.